volume/drivers/fake/fake_test.go

```diff
@@ -523,14 +564,14 @@ func TestFakeCloudBackupStateChange(t *testing.T) {
 	d, err := newFakeDriver(map[string]string{})
 	assert.NoError(t, err)
 
-	backupId, _, vol := testInitForCloudBackups(t, d)
+	taskId, _, _, vol := testInitForCloudBackups(t, d)
 
 	// Update element on db
 	var elem *fakeBackups
-	_, err = d.kv.GetVal(backupsKeyPrefix+"/"+backupId, &elem)
+	_, err = d.kv.GetVal(backupsKeyPrefix+"/"+taskId, &elem)
```

Comment:
Not related to your change but these tests seem weird/wrong. They shouldn't be modifying kvdb directly. There should be hooks in the fake driver for doing it.